graylog2-server/src/main/java/org/graylog2/rest/resources/system/ConfigurationResource.java

```diff
@@ -37,20 +36,17 @@
 @Path("/system/configuration")
 @Produces(MediaType.APPLICATION_JSON)
 public class ConfigurationResource extends RestResource {
-
     private final Configuration configuration;
-    private final ElasticsearchConfiguration esConfiguration;
 
     @Inject
-    public ConfigurationResource(Configuration configuration, ElasticsearchConfiguration esConfiguration) {
+    public ConfigurationResource(Configuration configuration) {
         this.configuration = configuration;
-        this.esConfiguration = esConfiguration;
     }
 
     @GET
     @Timed
     @ApiOperation(value = "Get relevant configuration settings and their values")
     public ExposedConfiguration getRelevant() {
-        return ExposedConfiguration.create(configuration, esConfiguration);
+        return ExposedConfiguration.create(configuration);
```

Comment:
This needs a note in the UPGRADING document since we are removing some data from the API endpoint.